modules/renter/download.go

```diff
@@ -295,13 +323,34 @@ func (r *Renter) addDownloadToChunkQueue(d *download) {
 			continue
 		}
 
+		// Calculate number of pieces to add to the download list.
+		// If the chunk's download speed should be sped up, the piece
+		// count will be raised above the minimum required.
+		// speedupchunk is true if it is the Nth last chunk where N is defined by `numFastChunks`.
```

Comment:
Looks like the comment formatting for all the bulk comments is off. If you use vim, you can get it correct easily by highlighting the block comment in visual mode, and then pressing `gq`. Sublime text has an equivalent, and most code-editors also have some sort of equivalent.